packages/core/src/useHiddenField/useHiddenField.ts

```diff
@@ -25,12 +24,15 @@ export interface HiddenFieldProps<TValue = unknown> {
 export function useHiddenField<TValue = unknown>(_props: Reactivify<HiddenFieldProps<TValue>>) {
   const props = normalizeProps(_props);
 
-  const field = useFormField({
-    label: '',
-    disabled: props.disabled,
-    path: props.name,
-    initialValue: toValue(props.value),
-  });
+  const field = useFormField(
+    {
+      label: '',
+      disabled: props.disabled,
+      path: props.name,
+      initialValue: toValue(props.value),
+    },
+    BuiltInControlTypes.Hidden,
+  );
 
```

Comment:
Since `useHiddenField` doesn't have a control and calls `useFormField` directly, we pass its type directly as well.